test/bluesky.auth.dpop.test.ts

```
@@ -0,0 +1,109 @@
+import { describe, expect, it } from 'vitest';
+import {
+  dpopAthFromAccessToken,
+  ecdsaDerSignatureToRawP256,
```

ESLint

[error] 'ecdsaDerSignatureToRawP256' is defined but never used.

(`@typescript-eslint/no-unused-vars`)

---

[error] Replace `·generatePkceVerifier,·isValidPkceVerifier,·pkceChallengeFromVerifier·` with `⏎··generatePkceVerifier,⏎··isValidPkceVerifier,⏎··pkceChallengeFromVerifier⏎`

(prettier/prettier)